bin/truffleruby.sh

```diff
@@ -59,13 +25,17 @@ while [ -h "$SELF_PATH" ]; do
 done
 
 root=$(dirname "$(dirname "$SELF_PATH")")
+# Used by jvm_args
 root_parent=$(dirname "$root")
 
 # TODO (pitr-ch 01-Mar-2017): investigate if we still need it
 if [ -n "$RUBY_BIN" ]; then
     exec "$RUBY_BIN" "$@"
 fi
 
+# Source values provided by mx at build time
+source "$root/tool/jvm_args"
```

Comment:
if it's not committed, should it be ignored?

Reply:
Indeed.
One related question is where should we place this file?
Under `bin/` would be natural but maybe it's odd to have non-executable files in `bin`?

Reply:
I would put it into mxbuild if ignored.

Reply:
> I would put it into mxbuild if ignored.

I think the path from the launcher to that file should not depend on whatever directory `mx` chooses to create for building, it should be independent.

Reply:
I think `mxbuild` is pretty stable. Now we are polluting tool directory with build data.

Reply:
I did some other changes to the generated file as well, I can add the move to it, if you have nothing against it.

Reply:
`mxbuild` itself should be fairly stable, true.
One concern is we might want to ship that file in a testing JVM-based distribution, and then shipping a `mxbuild` directory feels wrong.
IMHO, the Bash launcher should move to `tool/` because it's mostly used in development (and other cases should just link it in bin/). Then I think it'd make sense to have the data file as a sibling of the Bash launcher.
I'm not opposed to it, feel free to create a PR.

Reply:
https://github.com/oracle/truffleruby/pull/1202 for now, let's do the other changes when it comes to it.